R/model.R

```diff
@@ -31,18 +31,31 @@
 #'  * n_detect: number of humans with an infection detectable by microscopy between an inclusive age range at this timestep. This
 #' defaults to n_detect_730_3650. Other age ranges can be set with
 #' prevalence_rendering_min_ages and prevalence_rendering_max_ages parameters.
-#'  * n_severe: number of humans with a severe infection detectable by microscopy 
-#'  between an inclusive age range at this timestep. Age ranges can be set with
+#'  * p_detect: the sum of probabilities of detection by microscopy between an
```

Comment:
As the output now contains both, I wonder if it will help users to have an example (in vignette?) when to use one or the other. The nuance of these output types was not something I really remember clocking before, so some guidance will likely be needed.
I guess that users would in general use the smoother p_inc/p_detect outputs, unless they were trying to model "actual" population numbers or elimination questions - is that how you see it too?

Reply:
I think potentially useful yes and worth keeping seeing as they have been coded up already.

That output is very helpful. I think that with a litte explainer of how/why the black and grey differ and when to use which kind of output would be great.